test/jasmine/tests/transform_sort_test.js

```diff
@@ -273,14 +273,18 @@ describe('Test sort transform interactions:', function() {
         }
 
         function hover(gd, id) {
-            return new Promise(function(resolve) {
+            return new Promise(function(resolve, reject) {
                 gd.once('plotly_hover', function(eventData) {
                     delete gd._lastHoverTime;
                     resolve(eventData);
                 });
 
                 var pos = getPxPos(gd, id);
                 mouseEvent('mousemove', pos[0], pos[1]);
+
+                setTimeout(function() {
+                    reject('plotly_hover did not get called!');
```

Comment:
@alexcjohnson something like this?

e.g. if we comment out the above `mouseEvent()` we get:

![image](https://cloud.githubusercontent.com/assets/6675409/26462480/fbdda396-414e-11e7-8574-881934d59a9e.png)

from `fail_test.js`